lib/Elevate/RPM.pm

```diff
@@ -0,0 +1,84 @@
+package Elevate::RPM;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::RPM
+
+Logic wrapping the 'rpm' system binary
+
+=cut
+
+use cPstrict;
+
+use Log::Log4perl qw(:easy);
+
+use    # hide
```

Comment:
hide from what?

Reply:
I'm not actually sure here.  This file was a copy/paste of 'lib/Elevate/Service.pm' and this is how that file had it.  Not a big deal to change it.

Reply:
let's take it out. and even remove it from the other ones. I see no value and I think it's just going to cause cargo copying.

Reply:
done